examples/visual_recognition_v3.py

```diff
@@ -34,21 +33,21 @@
 #
 # cars_positive_examples=image_file), indent=2))
 
-url_result = visual_recognition.classify(images_url=test_url)
+url_result = visual_recognition.classify(parameters=json.dumps({'url': test_url}))
 print(json.dumps(url_result, indent=2))
 
-faces_result = visual_recognition.detect_faces(images_url=test_url)
+faces_result = visual_recognition.detect_faces(parameters=json.dumps({'url': test_url}))
 print(json.dumps(faces_result, indent=2))
 
 # print(json.dumps(visual_recognition.delete_classifier(classifier_id='YOUR
 # CLASSIFIER ID'), indent=2))
 
 print(json.dumps(visual_recognition.list_classifiers(), indent=2))
 
-file_path = join(dirname(__file__), '../resources/text.png')
-with open(file_path, 'rb') as image_file:
-    text_results = visual_recognition.recognize_text(images_file=image_file)
-    print(json.dumps(text_results, indent=2))
+#file_path = join(dirname(__file__), '../resources/text.png')
+#with open(file_path, 'rb') as image_file:
+#    text_results = visual_recognition.recognize_text(images_file=image_file)
+#    print(json.dumps(text_results, indent=2))
```

Comment:
Was this intended to be commented?

Reply:
Yes -- the text recognition methods of VR are deprecated.